libs/api/ebpf_api.cpp

```diff
@@ -2105,6 +2104,14 @@ _initialize_ebpf_programs_native(
     ebpf_result_t result = EBPF_SUCCESS;
 
     for (int i = 0; i < count_of_programs; i++) {
+        ebpf_program_t* program = programs[i];
+        if (!program->autoload) {
+#pragma warning(suppress : 6001) // The SAL annotation checks that program_handles[i] is ok.
+            if (program_handles[i] != ebpf_handle_invalid) {
+                Platform::CloseHandle(program_handles[i]);
```

Comment:
nit: I think we should set the handle to `ebpf_handle_invalid` here. Else any failure path after this can cause the handle to be closed twice.

Reply:
Done